install.sh

```diff
@@ -37,14 +37,31 @@ if ! which "wasm-merge" > /dev/null; then
     ARCH="x86_64"
   fi
 
-  curl -L -O "https://github.com/WebAssembly/binaryen/releases/download/$BINARYEN_TAG/binaryen-$BINARYEN_TAG-$ARCH-$OS.tar.gz"
+  if [ ! -e "binaryen-$BINARYEN_TAG-$ARCH-$OS.tar.gz" ]; then
+    echo 'Downloading binaryen...'
+    curl -L -O "https://github.com/WebAssembly/binaryen/releases/download/$BINARYEN_TAG/binaryen-$BINARYEN_TAG-$ARCH-$OS.tar.gz"
+  fi
+  rm -rf 'binaryen' "binaryen-$BINARYEN_TAG"
   tar xf "binaryen-$BINARYEN_TAG-$ARCH-$OS.tar.gz"
   mv "binaryen-$BINARYEN_TAG"/ binaryen/
-  sudo mkdir /usr/local/binaryen
-  sudo mv binaryen/bin/wasm-merge /usr/local/binaryen/wasm-merge
-  sudo ln -s /usr/local/binaryen/wasm-merge /usr/local/bin/wasm-merge
-  sudo ln -s /usr/local/binaryen/wasm-opt /usr/local/bin/wasm-opt
+  sudo mkdir -p /usr/local/binaryen
+  if ! which 'wasm-merge' > /dev/null; then
+    echo "Installing wasm-merge..."
+    rm -f /usr/local/binaryen/wasm-merge
+    sudo mv binaryen/bin/wasm-merge /usr/local/binaryen/wasm-merge
+    sudo ln -s /usr/local/binaryen/wasm-merge /usr/local/bin/wasm-merge
+  else
+    echo "wasm-merge is already installed"
+  fi
+  if ! which 'wasm-opt' > /dev/null; then
+    echo "Installing wasm-opt..."
+    rm -f /usr/local/bin/wasm-opt
+    sudo mv binaryen/bin/wasm-opt /usr/local/binaryen/wasm-opt
+    sudo ln -s /usr/local/binaryen/wasm-opt /usr/local/bin/wasm-opt
```

Comment:
perhaps we can address in another PR, one annoying thing i learned about this is not actually a full static binary. it some platforms expect to dynamically load the binaryen library. But if you have to install the binaryen library yourself then this step is kind of useless because you already have the binary.

Reply:
That is annoying, if we know what platforms, maybe it would be good to fallback to a wasm version in that case. I'd be hesitant to switch them all to wasm to avoid slowing down builds of plugins with the js-pdk.